src/NUnitFramework/framework/Attributes/RetryAttribute.cs

```diff
@@ -36,16 +36,16 @@
 namespace NUnit.Framework
 {
     /// <summary>
-    /// RepeatAttribute may be applied to test case in order
-    /// to run it multiple times.
+    /// RetryAttribute is used on a test method to specify that it should
+    /// be rerun if it fails, up to a maximum number of times.
     /// </summary>
     [AttributeUsage(AttributeTargets.Method, AllowMultiple = false, Inherited = false)]
     public class RetryAttribute : PropertyAttribute, IWrapSetUpTearDown
     {
         private int _count;
 
         /// <summary>
-        /// Construct a RepeatAttribute
+        /// Construct a RetryAttribute
         /// </summary>
         /// <param name="count">The number of times to run the test</param>
```

Comment:
We could also remove the same TODO in nunit/src/NUnitFramework/tests/Attributes/RetryAttributeTests.cs

Ps. You should remember to reference the issue number in the commitmessage or PR, like fixes #2313 such that the issue is automatically closed.